sportsdataverse/nfl/nfl_pbp.py

```
@@ -4190,6 +3916,392 @@ def __add_drive_data(self, play_df):
+                .then(pl.lit("punt"))
+                .when(pl.col("_ff_fg") == 1)
+                .then(pl.lit("field_goal"))
+                .when(pl.col("pass") == True)
+                .then(pl.lit("pass"))
+                .when(pl.col("rush") == True)
+                .then(pl.lit("run"))
+                .otherwise(pl.lit("no_play"))
+                .alias("_ff_play_type"),
+                # own_kickoff_recovery: kicking team retains a recovered kick
+                (
+                    (pl.col("kickoff_play") == True)
```

Flake8 (7.3.0)

[error] comparison to True should be 'if cond is True:' or 'if cond:'

(E712)